src/posts/votes.js

```diff
@@ -243,6 +245,13 @@ module.exports = function (Posts) {
 		if (!postData || !postData.pid || !postData.tid) {
 			return;
 		}
+		const threshold = meta.config['flags:autoFlagOnDownvoteThreshold'];
+		if (threshold && postData.votes <= (-threshold)) {
```

Comment:
Maybe change this to `if (threshold && postData.votes === -threshold)`
I believe right now it will go in the block and create a new flag for each downvote after it passes the threshold?

Reply:
This is by the spec, it adds a new "report" in the same "flag" for each additional downvote after the threshold exceeded.